install.sh

```diff
@@ -123,6 +123,18 @@ if is_vshasta_node; then
     deploy "${BUILDDIR}/manifests/vshasta.yaml"
 fi
 
+# Check if Rack Resiliency (RR) is enabled and respective setup/ config is done.
+function is_rr_enabled () {
+    python3 "${ROOTDIR}/lib/check_rr_enablement.py"
```

Comment:
This will result in script termination if script is run with `bash -e` or `set -e` occurs earlier in the script. Better rewrite it as 

    if python3 "${ROOTDIR}/lib/check_rr_enablement.py"; then
        return 0
    else
        return 1
    fi

or even

    python3 "${ROOTDIR}/lib/check_rr_enablement.py" && return 0 || return 1